WalletWasabi/ModuleInitializer.cs

```diff
@@ -0,0 +1,28 @@
+namespace WalletWasabi;
+using System.Collections.Concurrent;
+using System.Reflection;
+using NBitcoin;
+using System.Runtime.CompilerServices;
+
+class ModuleInitializer
+{
+	[ModuleInitializer]
+	internal static void PatchTestNet()
+	{
+		_ = Network.RegTest; // This is necessary to force the static members to be initialized
```

Comment:
this might get optimized away by the compiler (unused variable). A safer way to force static initialization is to use: `RuntimeHelpers.RunClassConstructor(typeof(Network).TypeHandle);`